### ISSUE TITLE
[New Command] Parse Microsoft Feedback Suggestions

### ISSUE THREAD

**Issue by awickham10** [sqlcollaborative/dbatools#3327]
## Feature Request
Be able to query the SQL Server suggestions/bugs/etc. on https://feedback.azure.com/forums/908035-sql-server. 

### Additional information
I did some research on how to use their API and have a draft for this function.

Parameters:
- Category (Bug/Documentation/Other/Setup and Deployment/Suggestions)
- Status (Under Review/Need Feedback/Unplanned/Planned/Started/Completed/Declined/Moved/Archived/Closed)
- State (Public/Closed) - Cannot be used in conjunction with Status
- Sort (Hot/Newest/Oldest/Votes)
- Limit - Limit the number of items returned

Output:
- ID - ID of suggestion
- Category - category 
- Status - status
- Votes - number of votes
- Title - title
- Author - name of author
- Link - link to full suggestion
- Created - creation date
- Closed - closed date

**Comment by wsmelton** [comment]
I don't think this one really fits in the module. It is more for a dedicated module itself because it is not directly related to managing SQL Server itself.

**Comment by wsmelton** [comment]
I can see it useful mind you, but in dedicated module for itself. Especially if their API allows you to "submit" feedback.
